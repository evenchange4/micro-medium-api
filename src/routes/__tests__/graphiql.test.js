const graphiql = require('../graphiql');

jest.mock('graphql-server-micro', () => ({
  microGraphiql: d => d,
}));

it('should return graphiql with endpointURL', () => {
  expect(graphiql('/graphql')).toMatchSnapshot();
});
