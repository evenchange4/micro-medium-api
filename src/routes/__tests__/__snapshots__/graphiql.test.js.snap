// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return graphiql with endpointURL 1`] = `
Object {
  "endpointURL": "/graphql",
  "operationName": "PostQuery",
  "query": "query PostQuery($username: String!, $limit: Int) {
  posts(username: $username, limit: $limit) {
    title
    firstPublishedAt
    url
    subtitle
  }
  user(username: $username) {
    username
    name
    bio
  }
}",
  "variables": Object {
    "limit": 100,
    "username": "evenchange4",
  },
}
`;
